README.md

```diff
@@ -81,6 +81,15 @@ skpm link path-to-local-plugin
 
 It will also ask you if you want to [disable the caching mechanism](http://developer.sketchapp.com/introduction/preferences/#always-reload-scripts-before-running) and force Sketch to always reload a Plugin’s code from disk (recommended when developing).
 
+
+#### Watching for changes
+
+To automatically rebuild and link your plugin as you make changes:
```

Comment:
Ah, I see. I was misunderstanding the fact that the `link` command only needs to happen once, and not after each change. Will fix.